assets/sass/components/_button.scss

```diff
@@ -10,22 +10,40 @@ $large-height: 24px;
 	vertical-align: middle;
 	touch-action: manipulation;
 	cursor: pointer;
+	font-weight: normal;
 	background-image: none; // Reset unusual Firefox-on-Android default style; see https://github.com/necolas/normalize.css/issues/214
+	&:link,
+	&:visited,
+	&:hover,
+	&:active {
+		text-decoration: none;
+	}
 }
 
 @mixin button-color(
 	$background-color,
 	$border-color: transparent,
-	$text-color: choose-contrast-color($background-color, $aplus-foreground, $aplus-background)
+	$text-color:
+		choose-contrast-color(
+			$background-color,
+			$aplus-foreground,
+			$aplus-background
+		)
```

Comment:
I think there could be comma here, but on the other hand, I don't think this function will see more arguments

Reply:
I don't have a strong feeling on the style, but I did this now.